static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -32,16 +30,6 @@ function SharedGroupHeader({group}: Props) {
               shortId={group.shortId}
               avatar={<ProjectBadge project={group.project} avatarSize={20} hideName />}
             />
-            {group.issueCategory === IssueCategory.PERFORMANCE && (
-              <FeatureBadge
```

Comment:
Seems unnecessary to have this beta flag since perf issues have been out for years